apidoc/Titanium/UI/WebView.yml

```diff
@@ -530,6 +532,7 @@ events:
     properties:
       - name: url
         summary: The URL of the web document that is stopped.
+        type: String
```

Comment:
There's also a hard-coded `message` String property. I think it always returns `"Webview did not load blacklisted url."`